app/lib/theme/widgets/AdjustableCalculatedValueWidget.tsx

```diff
@@ -0,0 +1,85 @@
+import { WidgetProps } from "@rjsf/core";
+import NumberFormat from "react-number-format";
+
+/*
+  to get the calculated value from formContext, we need to tweak the label of the field to match the calculated value in the formContext.
+  for example, if the label is "Total Funding Request", we need to change it to "totalFundingRequest" and then add `CalculatedValue` to the end of the label.
+*/
+export const getFormattedLabel = (fieldLabel: string) =>
+  fieldLabel.charAt(0).toLowerCase() + fieldLabel.slice(1).replace(/\s/g, "");
+
+export const AdjustableCalculatedValueWidget: React.FC<WidgetProps> = (
+  props
+) => {
+  const {
+    schema,
+    uiSchema,
+    id,
+    disabled,
+    label,
+    onChange,
+    value,
+    formContext,
+  } = props;
+
+  // If we are using this widget to show numbers as money, we can set `isMoney` to true in the uiSchema.
+  const isMoney = uiSchema?.isMoney;
+
+  const calculatedValue =
+    formContext[`${getFormattedLabel(label)}CalculatedValue`];
+
+  return (
+    <div>
+      {calculatedValue && (
+        <>
+          <div style={{ marginBottom: "1em" }}>
+            {isMoney ? "$" : ""}
+            {calculatedValue}
+          </div>
+          <label htmlFor={id}>{label} (Adjusted)</label>
+        </>
+      )}
+      <NumberFormat
+        thousandSeparator
+        fixedDecimalScale={isMoney}
+        id={id}
+        prefix={isMoney ? "$" : ""}
+        disabled={disabled}
+        className="adjustable"
+        decimalScale={isMoney ? 2 : 10} //Hardcoded for now, we can change it if we need to
+        defaultValue={(schema as any).defaultValue}
+        value={value ?? calculatedValue}
```

Comment:
We don't want to display the calculated value in this input, only an eventual adjusted value. If we put the calculated value here, it will end up in the db, and we won't know whether the user adjusted the calculation or not